monitoring/config-generator/src/generate.js

```diff
@@ -93,6 +113,17 @@ function prometheusConfig (env) {
       case 'kubernetes':
         const namespaces = (env.kubeNamespaces) ? env.kubeNamespaces.split(',') : null
         obj.scrape_configs = getPromKubeJobs(namespaces)
+        if (env.kubeLongterm) {
```

Comment:
This could be a `yargs` flag, yeah?

Reply:
yes, it can be either an env or `--kube-longterm` arg